app/panel/components/InsightsPromoModalContent.jsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * Insights Promo Modal Component
```

Comment:
Is this component not with the other modal content components because it is only used in the panel? Even if that's the case, I would opt for keeping them together in one `modal-content` directory in `shared-components`.

Reply:
The `PremiumPromoModal` is the only one that's being used in the hub. But yeah it makes more sense just to bag them all together in there too since it might be an option at one point. Moved!